plugins/CoreHome/angularjs/common/services/periods.js

```diff
@@ -384,4 +385,16 @@
         date.setMilliseconds(0);
         return date;
     }
+
+    function todayIsInRange(dateRange) {
```

Comment:
Wondering if it might be more useful to change that method so it directly works with period and date param. I guess in most places we are working with those params and not already with a parsed period.
Or alternatively we could add a method to all period classes instead, so in the end we would be able to call something like 
`$piwikPeriods.parse(currentPeriod, currentDate).containsToday()`

Reply:
Whatever this methods looks like in the end: It would be useful to add some simple tests for it to prevent any regressions.

Reply:
I thought about that and now that you mentioned it, I like the `$piwikPeriods.parse(currentPeriod, currentDate).containsToday()` call.
yeah, definitely a test is useful here.

Reply:
If I add the `containsToday()` to all period classes, should we add this as a breaking change or mention it anyway in the changelog? If someone has a custom range, but that doesn't have this method yet, there will be an error. Or should I check if the method is there? But in the future probably will forget to add the check anyway.
what should be the good solution here?

Reply:
Actually I doubt anyone is using custom period js classes at all. Also I don't think this part is public API. Guess we don't need to mention it in changelog 🤔   ping @tsteur

Reply:
we won't need to mention it in the changelog as it's not public API (almost nothing in our JS is public API) .Was thinking maybe cohorts did but doesn't look like it and it seems we're not using it in any of our plugins so nothing needed to be done 👍